knimin/tests/test_pm_create_study.py

```diff
@@ -0,0 +1,57 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import main
+from functools import partial
+import re
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db, qiita_client, jira_handler
+
+
+class TestPMCreatePlateHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get('/pm_create_study/')
+        self.assertEqual(response.code, 200)
+        self.assertTrue(
+            response.effective_url.endswith('?next=%2Fpm_create_study%2F'))
+
+    def test_get(self):
+        self.mock_login_admin()
+        response = self.get('/pm_create_study/')
+        self.assertEqual(response.code, 200)
+        self.assertIn('<h3>Create new project</h3>', response.body)
+        self.assertIn('LabDude (knight lab)</option>', response.body)
+
+    def test_post(self):
+        self.mock_login_admin()
+        data = {'qiita-user': 'demo@microbio.me',
```

Comment:
Can there be a creation failure as well? The underlying code is tested but it would be good to assert the result is still appropriately handled for the client